python/mxnet/visualization.py

```diff
@@ -339,11 +339,17 @@ def looks_like_weight(name):
         elif op == "BatchNorm":
             attr["fillcolor"] = cm[3]
         elif op in ('Activation', 'LeakyReLU'):
-            label = r"%s\n%s" % (op, node["attrs"]["act_type"])
+            if op == "LeakyReLU":
```

Comment:
if special handling is needed for each case, then the outer `elif op in ...` is not necessary.